src/Avalonia.NameGenerator/GeneratorOptions.cs

```diff
@@ -35,13 +35,15 @@ public Behavior AvaloniaNameGeneratorBehavior
         {
             get
             {
+                var defaultBehavior = Behavior.OnlyProperties;
+
                 var propertyValue = _context
                     .GetMSBuildProperty(
                         nameof(BuildProperties.AvaloniaNameGeneratorBehavior),
-                        nameof(Behavior.OnlyProperties));
+                        defaultBehavior.ToString());
 
                 if (!Enum.TryParse(propertyValue, true, out Behavior behavior))
-                    return Behavior.OnlyProperties;
+                    return defaultBehavior;
```

Comment:
What's the reasoning behind this particular change?

Reply:
in this case we set the default behavior in two places, this change allows us to set it once

Reply:
Worth doing the same to `AvaloniaNameGeneratorDefaultFieldModifier` to be consistent thanks.